windsurf/.windsurf/rules/geo-seo.md

```
@@ -0,0 +1,590 @@
+<\!-- trigger: model_decision -->
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)